clouddrift/analysis.py

```diff
@@ -1,7 +1,54 @@
 import numpy as np
+from collections.abc import Iterable
 from typing import Optional, Tuple
 import xarray as xr
+from concurrent import futures
 from clouddrift.haversine import distance, bearing
+from clouddrift.dataformat import unpack_ragged
+
+
+def apply_ragged(func, rowsize, arrs, *args, max_workers=None, **kwargs):
```

Comment:
What do you think about re-ordering `arrs` and `rowsize` in the function signature?

```suggestion
def apply_ragged(func, arrs, rowsize, *args, max_workers=None, **kwargs):
```

I think this is a more natural way to order it: We say something like "Apply `func` to `arrs` for every row of `rowsize`". It would also be more consistent with the `map` signature (`map(f, x)`).

Reply:
I don't have any preference.

Reply:
I prefer the order `arrs, rowsize`